File: 02 Searching Algorithms/Linear_Search.py
#program to implement binary search in a given array
import sys

size = int(input("Please enter the size of the list: "))
myList = list()
for i in range(0,size):
    myList.append(int(input("Please enter the element no {}: ".format(i+1))))

num = int(input("Please enter the number you want to search for: "))
for i in range(0,size):
    if myList[i]==num:
        print("Element exists in list at index {}".format(i))
        sys.exit() #Since the number has been found, no point of executing further
print("Element does not exist in list")
                

File: C_old_codes/Searching Algorithms/Linear_Search.c
#include <stdio.h>
int main(){
	int arr[] = {1,2,3,4,5,6,7,8,9,10};
	printf("Please enter the element you want to search for: ");
	int num = 0;
	scanf("%d",&num);
	int idx=-1, i=0;
	for(;i<10;i++){
		if(arr[i]==num) idx=i;
	}
	idx==-1?printf("\nElement not found!"):printf("\nElement found at index position: %d",idx);
	return 0;
}
